web/tests/realm_playground.test.js

```diff
@@ -76,14 +76,14 @@ run_test("get_pygments_typeahead_list_for_settings", () => {
     let iterator = candidates.entries();
     assert.equal(iterator.next().value[1], $t({defaultMessage: "Custom language: custom_lang"}));
     assert.equal(iterator.next().value[1], $t({defaultMessage: "Custom language: invent_a_lang"}));
-    assert.equal(iterator.next().value[1], "JavaScript (javascript, js, javascript, js)");
+    assert.equal(iterator.next().value[1], "JavaScript (javascript, js, javascript, and js)");
```

Comment:
Looking at these tests ... I think we are missing a deduplication+sorting step, as we shouldn't have `js` appear twice in a given list of what language ode generate something... or maybe we just have everything twice for some reason?

Want to add a bonus commit fixing that?

Reply:
Let's fix this in a new commit.

Reply:
Done.